src/rgw/rgw_process.cc

```diff
@@ -327,18 +328,29 @@ int process_request(const RGWProcessEnv& penv,
     abort_early(s, NULL, -ERR_METHOD_NOT_ALLOWED, handler, yield);
     goto done;
   }
+  is_health_or_system_request = (
+    (op->get_type() == RGW_OP_GET_HEALTH_CHECK) || s->user->get_info().admin ||
```

Comment:
is this going to cover multisite sync ops as well?

Reply:
yeah ,I think the `s->user->get_info().admin` contains the sync ops